src/components/header.jsx

```diff
@@ -28,14 +28,10 @@ function Header({ slug }) {
   const onClick = () => setExpanded((prevExpanded) => !prevExpanded);
 
   /* dynamically store parent links */
-  const parentLinks = links.parentLinks;
+  const { parentLinks } = links; // links + labels
   const parentLinksLength = parentLinks.length;
-  const parentLinksLabels = t('navigation.parentLinkLabels');
+  const parentLinksLabels = t('navigation.parentLinkLabels'); // labels
```

Comment:
Should the labels be coming from the links constants file instead of from the translation file?

Reply:
The labels are coming from the links constants file, but the translation file labels are used to check if the links and labels in the links constant files are consistent with the labels in the translation files.

Reply:
Oh okay, maybe the comment should be updated then? When I see these lines and the ` // labels` comment, I think that the labels from the translation file are being used

Reply:
Good point! Comment updated to `const parentLinksLabels = t('navigation.parentLinkLabels'); // labels (used for verification only; labels are taken from parentLinks)`